pipeline/base/task_terraform_run.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: terraform-run
+spec:
+  inputs:
+    resources:
+    - name: workspace
+      type: git
+      targetPath: infra
+    params:
+    - name: ops_or_apps
+      descriptions: Use ops- or apps terraform workspace.
+  volumes:
+    - name: kbst-pipeline-sa
+      secret:
+        secretName: kbst-pipeline-sa
+  steps:
+  - name: init
+    image: kubestack/cd:dev
+    command:
+    - terraform
+    args:
+    - init
+    envFrom:
+    - configMapRef:
+        name: env
+    workingDir: /workspace/infra
+    volumeMounts:
+    - name: kbst-pipeline-sa
+      mountPath: /secret
+    env:
+    - name: GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
WIP: Instead of using a K8s secret, it would be preferable to use the K8s service account as the Google IAM identity.